packages/circus-api/src/schemas/preferences.yaml

```
@@ -41,4 +41,11 @@ properties:
     type: array
     items:
       type: string
+  labelColors:
+    type: object
+    properties:
+      useDefault:
+        type: boolean
+      customColors:
+        type: array
```

これだと何の配列でもよいことになってしまいます。

```suggestion
        type: array
        items:
          type: string
          pattern: "^#[0-9a-f]{6}$"
```